zilencer/lib/stripe.py

```diff
@@ -16,6 +17,7 @@
 from zerver.models import Realm, UserProfile, RealmAuditLog
 from zilencer.models import Customer, Plan
 from zproject.settings import get_secret
+from confirmation.models import generate_key
```

Comment:
We have a number of functions like these in the codebase. Tim, do you have an opinion on which is the right one to import?